.gitignore

```diff
@@ -53,3 +53,4 @@ examples/
 build/dist
 scripts/aws/aws
 controllers/sriov-controller/sriov-controller
+cmd
```

Comment:
Why we need this?

Reply:
~3s
`<testcase classname="github.com/networkservicemesh/networkservicemesh/test/cloudtest/pkg/utils" name="TestProcessOutputShouldNotLostOutput" time="2.630000"/>` from JunitResult

Reply:
I have the untracked `cmd` binary in the repo after I did sh scripts/for-each-module.sh "go build ./..."

Reply:
It this my local problem?

Reply:
will check on another machine. Marked PR as WIP